dspace-spring-rest/src/main/java/org/dspace/app/rest/converter/BrowsableDSpaceObjectConverter.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.converter;
+
+import org.dspace.browse.BrowsableDSpaceObject;
+
+/**
+ * This is the base converter from/to objects in the DSpace API data model and
+ * the REST data model that can be indexed
+ *
+ * @param <M> the Class in the DSpace API data model
+ * @param <R> the Class in the DSpace REST data model
+ * @author Andrea Bollini (andrea.bollini at 4science.it)
+ */
+public abstract class BrowsableDSpaceObjectConverter<M extends BrowsableDSpaceObject,
+    R extends org.dspace.app.rest.model.RestAddressableModel> extends DSpaceConverter<M, R> {
+
+    public abstract boolean supportsModel(BrowsableDSpaceObject bdso);
```

Comment:
Please add Javadocs for this abstract method

Reply:
done https://github.com/DSpace/DSpace/pull/2312/files#diff-85f0e3672211829b8fceb596884bcdebR25